adapters/smaato/image_test.go

```diff
@@ -0,0 +1,33 @@
+package smaato
+
+import (
+	"testing"
+)
+
+func TestRenderAdMarkup(t *testing.T) {
+	type args struct {
+		adType             adMarkupType
+		adapterResponseAdm string
+	}
+	tests := []struct {
+		testName string
+		args     args
+		result   string
+	}{
+		{"imageTest", args{"Img", "{\"image\":{\"img\":{\"url\":\"//prebid-test.smaatolabs.net/img/320x50.jpg\"," +
```

Comment:
That's a lot of embedded horizontal scrolling. Is it possibly to simplify this a bit or load it from a separate json file?

Reply:
done